grunt/helpers/plugins/Plugin.js

```diff
@@ -96,6 +100,18 @@ class Plugin {
     return this.packageJSONFile.firstFileItem.item.version;
   }
 
+  /** @returns {string} */
+  get requiredFramework() {
+    return this.packageJSONFile.firstFileItem.item.framework;
+  }
+
+  /** @returns {boolean} */
+  get isFrameworkCompatible() {
+    if (!this.framework || !this.framework.version) return;
```

Comment:
Can we return `true` here and then apply a boolean check on line 66? (i.e. `!this.isFrameworkCompatible`)